### PR TITLE
Add transpilation to css-in-js-helpers package

### LINKED ISSUE #10: VS Code doesn't build properly
#7, and specifically https://github.com/ansumanshah/css-in-js/pull/7/commits/cba15acb9dd784a5e104cfca3fb00a25dcc44f68, introduced a bug for the build of the VS Code version. The build is now importing a common `css-in-js-helpers` package but it's not being transpiled through babel, throwing errors in the resulting build:

```
Activating extension `paulmolluzzo.convert-css-in-js` failed:  Unexpected token export
Here is the error stack:  /Users/molluzzp/css-in-js-package/packages/vscode-css-in-js/node_modules/css-in-js-helpers/index.js:28
export default function convert(s) {
^^^^^^
SyntaxError: Unexpected token export
```

The two possible solutions are:
1 - symlink the relevant files into the VS Code version and import them directly.
2 - include a transpilation/build step into the `css-in-js-helpers` package.

I'm inclined to go with the second option. @ansumanshah any thoughts or preferences?

### PR DESCRIPTION
**Pull request by paulmolluzzo**
Fixes #10 

Fixes the VS Code version build as VS Code doesn't support es2015 syntax.

### CODE REVIEW

File: packages/css-in-js-helpers/package.json [hunk 1]
```diff
@@ -16,5 +19,12 @@
   "bugs": {
     "url": "https://github.com/ansumanshah/css-in-js/issues"
   },
-  "homepage": "https://github.com/ansumanshah/css-in-js#readme"
+  "homepage": "https://github.com/ansumanshah/css-in-js#readme",
+  "devDependencies": {
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
```
Comment (ansumanshah) [inline packages/css-in-js-helpers/package.json]: I feel we dont need the loader and only one preset is required

**Comment by paulmolluzzo** [comment]
@ansumanshah I corrected according to those comments.

**Comment by ansumanshah** [comment]
Shall I merge this then or you are adding something else?

**Comment by paulmolluzzo** [comment]
You can merge if it looks right to you and doesn't break the atom version. 😀

I'll do the autocomplete in a separate PR. I just found this issue when I started this AM and wanted to correct it first.

**Pull request merged**